prow/cmd/checkconfig/main.go

```diff
@@ -123,6 +123,7 @@ const (
 	validateLabelWarning                          = "validate-label"
 	requiredJobAnnotationsWarning                 = "required-job-annotations"
 	periodicDefaultCloneWarning                   = "periodic-default-clone-config"
+	presubmitTestRegex                            = "(?mi)/test\\s.*"
```

Comment:
We should not define a constant regexp here. This needs to be the value from config (`AllowedPresubmitTriggerReRawString`) everywhere this is currently being used.